software/wordclock/wordclock.ino

```diff
@@ -10,7 +10,7 @@ RTC_DS3231 rtc;
 #ifdef NODO
 EnglishClockFace clockFace(ClockFace::LightSensorPosition::Top);
 // FrenchClockFace clockFace(ClockFace::LightSensorPosition::Top);
-// DutchClockFace clockFace(ClockFace::LightSensorPosition::Top);
+//DutchClockFace clockFace(ClockFace::LightSensorPosition::Top);

```

Comment:
Unnecessary change :)

Reply:
Its consistent now with the "French" clock and the other side of the if/else block.